composer.json

```diff
@@ -19,6 +19,7 @@
         "drupal/config_split": "^1.6",
         "drupal/core-composer-scaffold": "^9",
         "drupal/core-recommended": "^9",
+        "drupal/examples": "^3.0",
```

Comment:
we don;t need composer.lock changes?

Reply:
I'm following acquia/drupal-recommended-project. 
composer.lock file is ignored in that project repo.
https://github.com/acquia/drupal-recommended-project/blob/master/.gitignore#L2